core-java-modules/core-java-collections-maps-3/src/test/java/com/baeldung/map/hashmap/loadfactor/HashMapLoadFactorUnitTest.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.map.hashmap.loadfactor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+public class HashMapLoadFactorUnitTest {
+
+    @Test
+    public void whenCreateMapWithDefaultParam_thenSucces() {
+        Map<String, String> mapWithDefaultParams = new HashMap<String, String>();
+        mapWithDefaultParams.put("1", "One");
+        mapWithDefaultParams.put("2", "two");
+        mapWithDefaultParams.put("3", "three");
+        mapWithDefaultParams.put("4", "four");
+        mapWithDefaultParams.put("5", "five");
+        
+        Assert.assertEquals(5, mapWithDefaultParams.size());
+    }
+
+    @Test
+    public void whenCreateMapWithInitialCapacity_thenSucces() {
+        Map<String, String> mapWithInitialCapacity = new HashMap<String, String>(5);
```

Comment:
same as above -- no need to provide the types on the right as they are inferred from the declaration